vagrant/debian/Vagrantfile

```diff
@@ -25,34 +25,36 @@ hostname = "omv7box"
 Vagrant.configure("2") do |config|
 	config.vm.box = "debian/bookworm64"
 	config.vm.define hostname
-	config.vm.provision :shell, :path => "patch.sh"
-	config.vm.provision :reload
+
 	config.vm.provision :shell, :path => "install.sh"
 	config.vm.network "private_network", ip: "192.172.16.24", auto_config: false
 	config.vm.network "private_network", ip: "192.172.16.25"
 	config.vm.hostname = hostname
-	config.vm.provider :virtualbox do |prov|
+	
+	config.vm.provider :virtualbox do |prov, overwrite|
+		overwrite.vm.provision :shell, :path => "patch.sh", :args => ["enp0s3", "enp0s9"], reboot: true
+		
 		prov.name = hostname
 		prov.memory = "4096"
+		
 		# Add some disks.
-		for i in 0..2 do
-			filename = "./disks/#{hostname}-disk#{i}.vmdk"
-			unless File.exist?(filename)
-				prov.customize ["createmedium", "disk", "--filename", filename,
-					"--size", 1000 * 1024]
-				prov.customize ["storageattach", :id, "--storagectl",
-					"SATA Controller", "--port", i + 1, "--device", 0,
-					"--type", "hdd", "--medium", filename]
-			end
+		(0..2).each do |i|
+			overwrite.vm.disk :disk, size: "100GB", name: "#{hostname}-disk-#{i}"
 		end
 	end
-	config.vm.provider :libvirt do |prov|
+	
+	config.vm.provider :libvirt do |prov, overwrite|
+		overwrite.vm.provision :shell, :path => "patch.sh", :args => ["ens6", "ens8"]
+		overwrite.vm.provision :reload
```

Comment:
This does not work for me:
```
$ vagrant --version
Vagrant 2.4.1
$ vagrant plugin list                          
vagrant-libvirt (0.12.2, global)
$ vagrant up
Bringing machine 'omv7box' up with 'libvirt' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* The 'reload' provisioner could not be found.
```

Reply:
Strange. You do still have the reload plugin installed?

Reply:
Yes
```
$ vagrant --version
Vagrant 2.4.1
$ vagrant plugin install vagrant-reload
Installing the 'vagrant-reload' plugin. This can take a few minutes...
Fetching nokogiri-1.16.4-x86_64-linux.gem
Fetching ruby-libvirt-0.8.2.gem
Building native extensions. This could take a while...
Fetching vagrant-reload-0.0.1.gem
Removing nokogiri
Successfully uninstalled nokogiri-1.16.4-x86_64-linux
Successfully uninstalled ruby-libvirt-0.8.2
Installed the plugin 'vagrant-reload (0.0.1)'!
$ vagrant up                           
Bringing machine 'omv7box' up with 'libvirt' provider...
==> omv7box: Checking if box 'debian/bookworm64' version '12.20231211.1' is up to date...
==> omv7box: Creating image (snapshot of base box volume).
==> omv7box: Creating domain with the following settings...
==> omv7box:  -- Name:              debian_omv7box
==> omv7box:  -- Description:       Source: /home/vot/git/openmediavault/vagrant/debian/Vagrantfile
==> omv7box:  -- Domain type:       kvm
==> omv7box:  -- Cpus:              1
==> omv7box:  -- Feature:           acpi
...
```

Reply:
Why did you switch from the default reboot shell option to the reload plugin in the past? Is it still needed for libvirt to begin with? 

Would you mind testing this by commenting out the reload provisioner line and adding `, reboot: true` at the end of the patch shell line? Just like with the VB provider.

The reload plugin dependency could be removed all together if this works now.